frontend/src/_helpers/postgrestQueryBuilder.js

```diff
@@ -132,9 +132,14 @@ export default class PostgrestQueryBuilder {
    * @param value  The value to filter with.
    */
   is(column, value) {
-    this.url.append(`${column}`, `is.${value}`);
+    if (value === 'notNull') {
```

Comment:
Is this the best way to do it is not null operation? open for feedback. :)

Reply:
This is fine since we don't have a not operator to use in conjunction.